tests/pipeline/test_estimate.py

```
@@ -0,0 +1,167 @@
+
+        # Extract ATE value from "ATE: ..." format
+        if not content.startswith("ATE:"):
+            print(f"❌ Error: Expected format 'ATE: ...' in {ate_file}, got: {content}")
+            return None
+
+        ate_str = content.replace("ATE:", "").strip()
+        true_ate = float(ate_str)
+
+        return true_ate
+
+    except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)